src/graph_notebook/magics/streams.py

```diff
@@ -0,0 +1,225 @@
+import re
+import json
+import urllib.request
```

Comment:
Nit: Can remove this line and the next since we no longer use `urllib` here

Reply:
Fixed